src/Microsoft.DotNet.Internal.Tools.SynchronizePackageProps/Program.cs

```diff
@@ -30,6 +31,10 @@ private static async Task<Errors> Process(string[] args)
                     "directory|dir|d=", "Directory of eng folder containing Packages.props and Version.Details.xml",
                     v => dir = v
                 },
+                {
+                    "source-directory|src-dir|s=", "Directory of the src folder containing Directory.Packages.props",
```

Comment:
Its probably better to just accept a single directory that is the repo root, and search for both files from there.